pkg/build/api/v1beta1/conversion_test.go

```diff
@@ -759,3 +759,77 @@ func TestContextDirConversion(t *testing.T) {
 		t.Errorf("expected %v, actual %v", oldVersion.Strategy.DockerStrategy.ContextDir, actual.Source.ContextDir)
 	}
 }
+
+func TestBuildTriggerPolicyOldToNewConversion(t *testing.T) {
+	testCases := map[string]struct {
+		Old                      older.BuildTriggerPolicy
+		ExpectedBuildTriggerType newer.BuildTriggerType
+	}{
+		"ImageChange": {
```

Comment:
These are just arbitrary test labels used when something goes wrong (see [1](https://github.com/openshift/origin/pull/2750/files#diff-994e87ad6ef744a8680c3b6981c995efR792) & [2](https://github.com/openshift/origin/pull/2750/files#diff-994e87ad6ef744a8680c3b6981c995efR795)).

Same thing as in many other places that use `testCases := map[string]struct`, e.g. [pkg/template/template_test.go](https://github.com/openshift/origin/blob/82458ada79005120d9436cf9dbe1a2f0894101bd/pkg/template/template_test.go#L154-L310).